### ISSUE TITLE
sdwan_cisco_vpn_interface_feature_template "nat" value issue

### ISSUE THREAD

**Issue by suitesss** [CiscoDevNet/terraform-provider-sdwan#102]
My vManage is 20.12.1
sdwan_cisco_vpn_interface_feature_template's "nat" default value is "true", and it can't be change to "false"
Thanks

**Comment by danischm** [comment]
Thanks for raising the issue! We haven't yet tested the provider with 20.12. Do you have a chance to test it with 20.9 to confirm it behaves the same?

**Comment by suitesss** [comment]
I test it with 20.9.3.2, it behaves the same.
I also test : using vManage GUI to change nat value to Off, then using Terraform to change it to On, it don't have any change.
Thanks

**Comment by danischm** [comment]
Thanks again for raising the issue! It looks like this was introduced with a change in 0.2.10. As a workaround you can pin the provider version to 0.2.9. We will work on a fix in the meantime.

**Comment by andreasbaekdahl** [comment]
I tried with version 0.2.10, 0.2.9 and 0.2.7. None of them are working

**Comment by andreasbaekdahl** [comment]
```hcl
terraform {
  required_providers {
    sdwan = {
      source  = "CiscoDevNet/sdwan"
      version = "0.2.9"
    }
  }
}
resource "sdwan_cisco_vpn_interface_feature_template" "example" {
  description = "GigabitEthernet0"
  device_types = [
    "vedge-C8000V"
  ]
  interface_name = "GigabitEthernet0"
  name           = "TF-VPN_INTERFACE_TEST"
  nat            = false
}
```

**Comment by danischm** [comment]
https://github.com/CiscoDevNet/terraform-provider-sdwan/commit/12dde0ee1abca0bb6261c1a59293b7c00ba74a14#diff-430dad7648335a8b6e1ddadd83a2f60d156851eb62331c68a581c1cc6ab92541

**Comment by danischm** [comment]
Fixed in v0.2.11 release.

**Comment by suitesss** [comment]
It works, Thanks
